src/Altinn.Apps/AppTemplates/AspNet/Altinn.App.PlatformServices/Implementation/AppResourcesSI.cs

```diff
@@ -92,14 +82,7 @@ public byte[] GetAppResource(string org, string app, string resource)
         /// <inheritdoc />
         public byte[] GetText(string org, string app, string textResource)
         {
-            byte[] fileContent = null;
-
-            if (File.Exists(_settings.AppBasePath + _settings.ConfigurationFolder + _settings.TextFolder + textResource))
-            {
-                fileContent = File.ReadAllBytes(_settings.AppBasePath + _settings.ConfigurationFolder + _settings.TextFolder + textResource);
-            }
-
-            return fileContent;
+            return ReadFileContentsFromLegalPath(_settings.AppBasePath + _settings.ConfigurationFolder + _settings.TextFolder, textResource);
```

Comment:
Is there a reason this is done through the new method but methods below is not.  Like GetRuleHandlers

Reply:
I focused on those that had untrusted input.